packages/solidity-contracts/hardhat.config.ts

```diff
@@ -48,6 +49,16 @@ const config: HardhatUserConfig = {
       url: `https://sepolia.infura.io/v3/${INFURA_API_KEY}`,
       accounts: CONTRACTS_DEPLOYER_KEY ? [CONTRACTS_DEPLOYER_KEY] : [],
     },
+    beta5: {
+      url: RPC_URL,
+      accounts: CONTRACTS_DEPLOYER_KEY
+        ? [CONTRACTS_DEPLOYER_KEY]
+        : {
+            mnemonic:
+              'test test test test test test test test test test test junk',
```

Comment:
test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test :)

Reply:
Well known mnemonic used by hardhat and anvil :)